server/routes/events.js

```diff
@@ -19,19 +19,17 @@ router.get('/', (req, res) => {
 })
 
 router.post('/:id/attending', getTokenDecoder(), (req, res) => {
-  if (req.user) {
-    const id = Number(req.params.id)
-    const event = Number(req.body.event)
-    addUserToEvent(id, event)
-      .then((ids) => {
-        res.status(200).json(ids[0])
-      })
-      .catch((err) => {
-        res.status(500).send(`DATABASE ERROR ${err.message}`)
-      })
-  } else {
-    res.status(500).send('authentication token not provided')
-  }
+  console.log('server')
+
+  const id = Number(req.params.id)
+  const event = Number(req.body.event)
+  addUserToEvent(id, event)
+    .then((ids) => {
+      res.status(200).json(ids[0])
+    })
+    .catch((err) => {
+      res.status(500).send(`DATABASE ERROR ${err.message}`)
+    })
```

Comment:
❕ To join an event, I think it makes sense to require auth — otherwise anyone with Postman could sign up any user to any event.  If we're getting auth issues when sending the API request, I think we need to figure out how to fix the API request, not remove the auth requirement.